java/src/main/java/com/google/oak/remote_attestation/AmdAttestationVerifier.java

```diff
@@ -50,11 +52,12 @@ public AmdAttestationVerifier(byte[] referenceValue) {
    */
   // TODO(#3641): Rewrite java-doc to represent actual AMD attestation verification.
   @Override
-  public final Result<Boolean, Exception> verify(
-      final AttestationEvidence evidence, final AttestationEndorsement endorsement) {
+  public final Result<AttestationResults, Exception> verify(long nowUtcMillis,
```

Comment:
I think this file (`AmdAttestationVerifier.java`) is outdated, because AMD vs Intel will be defined purely by Reference Values. So it could be renamed to `DefaultAttestationVerifier.java` (not sure it's the best name, because "default" is a scary word for security) or `CustomAttestationVerifier.java` (since it can accept any Reference Values)

Reply:
Deleted it for now, I guess the JNI wrapper will be the default going forward